test_param_tuple_arg1-error.chpl:7: error: unresolved call 'foo(3, 2*int(64))'
test_param_tuple_arg1-error.chpl:1: note: candidates are: foo(param rank: int, x: rank*int)
